commit 9be180ab1ac0f7e84818ae17c7fddd1f168d22da
Author: Fabio M. Di Nitto <fdinitto@redhat.com>
Date:   Tue Aug 23 09:25:52 2011 +0200

    cman: make RRP mode passive the default
    
    RRP in active mode is still too fresh and needs some more work in corosync.
    Put rrp_mode=passive as default and still allow users to tune it if they feel brave
    enough.
    
    Resolves: rhbz#732635
    
    Reviewed-by: Jan Friesse <jfriesse@redhat.com>
    Signed-off-by: Fabio M. Di Nitto <fdinitto@redhat.com>

diff --git a/cman/daemon/cman-preconfig.c b/cman/daemon/cman-preconfig.c
index 7374fa4..6d52be1 100644
--- a/cman/daemon/cman-preconfig.c
+++ b/cman/daemon/cman-preconfig.c
@@ -999,7 +999,7 @@ static void add_cman_overrides(struct objdb_iface_ver0 *objdb)
 		if (objdb_get_string(objdb, object_handle, "rrp_mode", &value)) {
 			if (num_interfaces > 1) {
 				objdb->object_key_create_typed(object_handle, "rrp_mode",
-							       "active", strlen("active")+1, OBJDB_VALUETYPE_STRING);
+							       "passive", strlen("passive")+1, OBJDB_VALUETYPE_STRING);
 			}
 			else {
 				objdb->object_key_create_typed(object_handle, "rrp_mode",
